scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala

```diff
@@ -57,7 +57,8 @@ class TypeProviderTestScala211 extends FlatSpec with Matchers {
     """.stripMargin)
   class ArtisanalMoreThan22Fields
 
-  "BigQueryType.fromSchema" should "support .schema in companion object with more than 22 fields" in {
+  "BigQueryType.fromSchema".should("support .schema in companion object with more than 22 fields")
```

Comment:
Shorten the 2nd string so we don't have to line break and reduce readability?
`with >22 fields`?